0029-stream/0029-stream.md

```diff
@@ -170,19 +172,28 @@ Incoming packets can either be associated with an existing connection, or, for s
 
 STREAM packets are completely encrypted so endpoints must try to decrypt and parse them to determine whether a given packet was sent by the other endpoint of a connection. Incoming Prepare packets whose data cannot be decrypted with the expected shared secret MUST be rejected with `F06: Unexpected Payment` errors.
 
-### 4.3. Client Address Communication and Connection Migration
+### 4.3. Optional Connection Details
+ 
+#### 4.3.1. Client Address Communication (Receivers Only)
 
-When a client connects to a server, they MUST communicate their ILP address to the server using a `ConnectionNewAddress` frame.
+When a client connects to a server, the client MAY communicate its ILP Address to the server using a  `ConnectionNewAddress` frame. This allows the client to function as a receiver on the Connection. Without this frame, a server would not have a destination address to send packets to a client.
 
-Either endpoint MAY change their ILP address at any point during a connection by sending a `ConnectionNewAddress` frame. To ensure the new address is received and acknowledged, implementations MAY choose to send these frames only in ILP Prepare packets.
+In the most advanced scenarios where clients and servers are both capable of simultaneously sending and receiving, the `ConnectionNewAddress` frame enables bi-directional payment flows. However, the frame is OPTIONAL because clients are commonly not capable of receiving. For example, a client may not be accessible from a routable ILP Address.
```

Comment:
```suggestion
In the most advanced scenarios where clients and servers are both capable of simultaneously sending and receiving, the `ConnectionNewAddress` frame enables bi-directional payment flows. However, the frame is OPTIONAL because clients are commonly not capable of receiving. For example, a client using a request-response protocol like HTTP to send ILP packets doesn't have a persistent connection for receiving ILP prepares.
```

Reply:
Accepted via https://github.com/interledger/rfcs/pull/573/commits/f3559431b55efdf57d8b3bf4acd81f22a47c925c